html/elements/input/color.json

```diff
@@ -0,0 +1,153 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "color": {
```

Comment:
I think I would nest this like @wbamberg proposed here: https://github.com/mdn/browser-compat-data/pull/1698#issuecomment-390289031

So, you'd have 
- html
  - elements
    - input
      - input-color
        - __compat
          -  mdn_url: https://developer.mozilla.org/docs/Web/html/Element/input/color
          - description : "type=\"color\"
          - support: { ....
        - list
        - autocomplete

Reply:
So basically "input-color" would be better, because "color" looks like a normal attribute, which it isn't.

Reply:
Gotcha! That all sounds good - thanks :)